api/src/documents.py

```diff
@@ -323,17 +324,38 @@ def latest_documents(scroll_id: Optional[str]=None, size: int = 10):
 @router.get(
     "/documents", response_model=DocumentSchema.DocumentListResponse
 )
-def list_documents(scroll_id: Optional[str]=None, size: int = 10):
+def list_documents(
+    scroll_id: Optional[str] = None,
+    size: int = 10,
+    sort_by: Optional[str] = Query(None, description="Field to sort by"),
+    order: Optional[str] = Query(None, description="Order to sort by")
+):
     """
     Retrieves all stored documents, regardless of files uploaded or processed.
+    Optional `sort_by` and `order` params return sorted results.
     """
 
+    if order not in ["asc", "desc"]:
```

Comment:
This will be a problem when using the API directly (which Uncharted's causemos app does).
sort_by and order are optional, but then order is required on this line. We should provide defaults for `sort_by` and `order`, and only validate `sort_by` was provided.

Reply:
Yeah good catch - I was just testing this with no `sort_by` or `order` and it currently fails.